### PR TITLE
File uploading guide

### PR DESCRIPTION
**Pull request by DenJohX**

### CODE REVIEW

File: guides/file-uploading.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    ]
+});
+```
+
+*Et voilà!*. Now we have a Feahtersjs file storage service working, with support for traditional multipart uploads, and a variety of storage options to choose. 
+
+**Simply awesome.**
+
+
+# Furhter improvements
```
Comment (wkw) [inline guides/file-uploading.md]: Typo: "Further"

File: guides/file-uploading.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+Recently we where struggling to find a way to upload files whitout having to write a separate Express middleware or having to (re)write a complex Feathers service.
+
+# Our Goals
+We want to implement an upload service in a way that it acomplishes these few important things:
+
+1. It has to handle large files (+10MB).
+2. Needs to work with the app's authetication and authorization.
+3. The files has to be validated.
+4. By the moment there is no third party storage service involved, but this will change in the near future, so it has to be prepared.
```
Comment (aminmeyghani) [inline guides/file-uploading.md]: grammar: `At the moment` instead of `By the moment`.

File: guides/file-uploading.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
+# File uploads in Feathersjs
```
Comment (aminmeyghani) [inline guides/file-uploading.md]: Very helpful guide! 👍

**Comment by daffl** [comment]
This is pretty awesome! If you also add the link to the "Guides" section in https://github.com/feathersjs/feathers-docs/blob/master/SUMMARY.md we can merge it right away!

Oh and this will close #111

**Comment by daffl** [comment]
Thanks again! I merged the PR and added the guide to the summary as well as made the two tweaks suggested in the comments. This is great!

**Comment by daffl** [comment]
You can find the guide at http://docs.feathersjs.com/guides/file-uploading.html

**Comment by DenJohX** [comment]
@daffl sorry, I had no time today to make the changes, also thank you! It feels great being accepted in the official docs.

**Comment by nikolakanacki** [comment]
For anyone landing here, the new guide link is https://docs.feathersjs.com/guides/advanced/file-uploading.html

**Pull request merged**